ibm/provider/provider.go

```diff
@@ -1248,6 +1249,7 @@ func Provider() *schema.Provider {
 			"ibm_iam_policy_template":                      iampolicy.ResourceIBMIAMPolicyTemplate(),
 			"ibm_iam_policy_template_version":              iampolicy.ResourceIBMIAMPolicyTemplateVersion(),
 			"ibm_iam_policy_assignment":                    iampolicy.ResourceIBMIAMPolicyAssignment(),
+			"ibm_iam_access_management_account_settings":   iampolicy.ResourceIBMIAMAccessManagementAccountSettings(),
 
```

Comment:
Can we rename this res/ds to `ibm_iam_access_settings` or something similar to name what we expose in cli command if present

Reply:
We will do `ibm_iam_account_settings_external_interaction` to match CLI: `ibmcloud iam account-settings-external-interaction-update`